pycloudflare/models.py

```diff
@@ -191,36 +194,36 @@ def __repr__(self):
 
 class Record(object):
     _data = ()
+    _own_attrs = ('zone', '_service', '_data', '_old_data')
 
     def __init__(self, zone, data):
         self.zone = zone
         self._service = zone._service
         self._data = data
-        self._updates = {}
+        self._old_data = deepcopy(data)
 
     def __getattr__(self, name):
-        if name in self._updates:
-            return self._updates[name]
         if name in self._data:
             return self._data[name]
+        if name in self._old_data:
+            return self._old_data[name]
```

Comment:
Why do we return anything from `_old_data`? `_data` is initialized with everything.